Makefile

```diff
@@ -6,7 +6,7 @@ MAKEFLAGS += -r
 .SUFFIXES:
 SHELL := /bin/bash
 
-include VERSION
+include VERSION container_images.mk
```

Comment:
Maybe one prefers 2 distinct includes ?

Reply:
#603 may change this. I think distinct `include` lines is a bit cleaner, but no blocker for me.